pkg/motor/object_builder_test.go

```diff
@@ -0,0 +1,77 @@
+package motor
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"testing"
+
+	st "github.com/sonr-io/sonr/x/schema/types"
+	"github.com/stretchr/testify/assert"
+	prt "go.buf.build/grpc/go/sonr-io/motor/api/v1"
+)
+
+/*
+prt.CreateSchemaRequest{
+	Label: "TestUser",
+	Fields: map[string]prt.CreateSchemaRequest_SchemaKind{
+		"email":     prt.CreateSchemaRequest_SCHEMA_KIND_STRING,
+		"firstName": prt.CreateSchemaRequest_SCHEMA_KIND_STRING,
+		"age":       prt.CreateSchemaRequest_SCHEMA_KIND_INT,
+	},
+}
+*/
+const SCHEMA_DID string = "did:snr:f2604d3b-4a30-4223-ba63-070a075b7db8"
```

Comment:
Can I run this in a pipeline?

Reply:
Yes so long as the chain state doesn't get removed. I added TODO for making a better test suite